common-test.dotty/src/main/scala/org/scalatest/SharedHelpers.scala

```diff
@@ -1802,18 +1804,18 @@ object SharedHelpers extends Assertions with LineNumberHelper {
   def sortedSet[T](elements: T*): SortedSet[T] = {
     val orderMap = Map.empty[T, Int] ++ elements.zipWithIndex
     val comparator = orderMapComparator(orderMap)
-    implicit val ordering: Ordering[T] = new Ordering[T] {
-      def compare(x: T, y: T): Int = comparator.compare(x, y)
-    }
+    implicit val ordering: Ordering[T] = (new Ordering[T | Null] {
+      def compare(x: T | Null, y: T | Null): Int = comparator.compare(x, y)
+    }).asInstanceOf[Ordering[T]]
     SortedSet.empty[T] ++ elements
   }
 
   def sortedMap[K, V](elements: (K, V)*): SortedMap[K, V] = {
     val orderMap = Map.empty[K, Int] ++ elements.map(_._1).zipWithIndex
     val comparator = orderMapComparator(orderMap)
-    implicit val ordering: Ordering[K]  = new Ordering[K] {
-      def compare(x: K, y: K): Int = comparator.compare(x, y)
-    }
+    implicit val ordering: Ordering[K]  = (new Ordering[K | Null] {
+      def compare(x: K | Null, y: K | Null): Int = comparator.compare(x, y)
```

Comment:
**Overridden java method arguments are nullable**
A Scala method overrides some Java method. We need to add `|Null` to the method's parameter types to make the override still work.